src/staff_rename.lua

```diff
@@ -163,40 +213,39 @@ function staff_rename()
         dialog:SetTitle(str)
 
         local row = {}
-        for i = 1, 100 do
+        for i = 1, (staff_count + 5) do
             row[i] = (i -1) * row_h
         end
 --
         local col = {}
-        for i = 1, 11 do
+        for i = 1, 5 do
             col[i] = (i - 1) * col_w
             col[i] = col[i] + 40
         end
 --
-
         function add_ctrl(dialog, ctrl_type, text, x, y, h, w, min, max)
             str.LuaString = text
-            local ctrl
+            local ctrl = ""
             if ctrl_type == "button" then
-                ctrl = dialog:CreateButton(x, y)
+                ctrl = dialog:CreateButton(x, y + 2)

```

Comment:
I don't think it's incredibly important to fix here, but this `2` is what's called a "magic number". A magic number is essentially just a hard-coded number like this where there's no real description of where it comes from. And with this number being the exact value it is, things somehow work like magic.

It's best to avoid magic numbers whenever possible. Though admittedly, here it makes sense. You're just increasing the vertical spacing by 2 pixels.

Here's a discussion link from one of Carl Vine's PRs about magic numbers:

https://github.com/finale-lua/lua-scripts/pull/314#discussion_r936174719

Reply:
I totally understand what you're saying... And I wish I didn't need the magic numbers to make the UI look good, but the elements just don't line up without them. 

One of these days I'm going to do a super comprehensive comparison between the way MacOS and Windows draw everything and how they line up, and try to do this sort of thing more systematically...